### PR TITLE
Feat/history liked

### PR DESCRIPTION
**Pull request by Aditi0505**
# Change Logs:

## Description:

- Added Liked Video Page and History.
- Added Liked Video and History functionality on the Single Video page.
- Added a few fixes.

## Screenshot 
image
image

# Preview Link
https://deploy-preview-5--starlightapp.netlify.app/

### CODE REVIEW

**Review by arpit01923 — APPROVED** [review]
- clean code, functionality works fine
- good naming convention
- good to merge

File: src/redux/features/videos/SingleVideo.js [hunk 1]
```diff
@@ -1,27 +1,49 @@
...
-  }, [videoId, dispatch]);
+    (async () => {
+      try {
+        const response = await axios.get(`/api/video/${videoId}`);
+        setCurrentVideo(response.data.video);
+      } catch (e) {
+        return e;
+      }
+    })();
+  }, [videoId]);
```
Comment (vishalg8454) [inline src/redux/features/videos/SingleVideo.js:34]: why not using the videos from videosSlice?

**Review by vishalg8454 — APPROVED** [review]
All looks good to merge. Left one suggestion.

File: src/redux/features/videos/SingleVideo.js (hunk 1 above)
Comment (Aditi0505) [reply src/redux/features/videos/SingleVideo.js:34]: This is the call to get a single video based on id. Can you please let me know what is the context of using videoList here?

Comment (vishalg8454) [reply src/redux/features/videos/SingleVideo.js:34]: Ah yes, I was mistaken. You are doing it correct.

**Pull request merged**